src/HDInsight/HDInsight/ManagementCommands/GrantAzureHDInsightRdpServicesAccessCommand.cs

```diff
@@ -16,11 +16,13 @@
 using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
 using Microsoft.Azure.Management.HDInsight.Models;
 using Microsoft.WindowsAzure.Commands.Common;
+using Microsoft.WindowsAzure.Commands.Common.CustomAttributes;
 using System;
 using System.Management.Automation;
 
 namespace Microsoft.Azure.Commands.HDInsight
 {
+    [CmdletDeprecation("3.0.0")]
```

Comment:
You need to provide a good, customer-facing string here.  What should the customer do if this cmdlet is not here to grant access?  Is there a different mechanism?  Is RDP service being discontinued?  The customer needs to be able to use this string to figure out how to change their script.

Reply:
> You need to provide a good, customer-facing string here. What should the customer do if this cmdlet is not here to grant access? Is there a different mechanism? Is RDP service being discontinued? The customer needs to be able to use this string to figure out how to change their script.

Fixed.